testutils/src/main/java/com/microsoft/identity/internal/testutils/TestUtils.java

```diff
@@ -72,12 +75,8 @@ public static SharedPreferencesFileManager getSharedPreferences(final String sha
      */
     public static SharedPreferencesFileManager getEncryptedSharedPreferences(final String sharedPrefName) {
         final Context context = ApplicationProvider.getApplicationContext();
-        final SharedPreferencesFileManager barePreferences = SharedPreferencesFileManager.getSharedPreferences(
-                context,
-                sharedPrefName,
-                Context.MODE_PRIVATE,
-                new AndroidCommonComponents(context).
-                        getStorageEncryptionManager(null));
+        final StorageHelper storageHelper = new StorageHelper(context);
+        final SharedPreferencesFileManager barePreferences = SharedPreferencesFileManager.getSharedPreferences(context, sharedPrefName, storageHelper);
```

Comment:
I think we need the code that was there on dev

Reply:
half of it.

Reply:
```suggestion
       final SharedPreferencesFileManager barePreferences = SharedPreferencesFileManager.getSharedPreferences(
           context,
           sharedPrefName,
           new AndroidCommonComponents(context).
                             getStorageEncryptionManager(null))
      );
```